sphinx/content/ch1.md

```diff
@@ -0,0 +1,1249 @@
+# Elegant NumPy: The Foundation of Scientific Python
+
+> [NumPy] is everywhere. It is all around us. Even now, in this very room.
+> You can see it when you look out your window or when you turn on your
+> television. You can feel it when you go to work... when you go to church...
+> when you pay your taxes.
+>
+> — Morpheus, *The Matrix*
+
+This chapter touches on some statistical functions in SciPy, but more than that, it focuses on exploring the NumPy array, a data structure that underlies almost all numerical scientific computation in Python.
+We will see how NumPy array operations enable concise and efficient code when manipulating numerical data.
+
+Our use case is using gene expression data from The Cancer Genome Atlas (TCGA) project to predict mortality in skin cancer patients.
+We will be working toward this goal throughout this chapter and the next one, learning about some key SciPy concepts along the way.
+Before we can predict mortality, we will need to normalize the expression data using a method called RPKM normalization.
+This allows the comparison of measurements between different samples and genes.
+(We will unpack what "gene expression" means in just a moment.)
+
+Let's start with a code snippet to tantalize you and introduce the ideas in this chapter.
+As we will do in each chapter, we open with a code sample that we believe epitomizes the elegance and power of a particular function from the SciPy ecosystem.
+In this case, we want to highlight NumPy's vectorization and broadcasting rules, which allow us to manipulate and reason about data arrays very efficiently.
+
+
+```{code-block} python
```

Comment:
Why aren't code-blocks the default?

Reply:
I believe three back-ticks without a {} section are interpreted as code blocks (per commonmark), but if you want to specify additional meta-data, then you have to explicitly use the {code-block} directive

Reply:
Is there a way/still time to keep this consistent with GH-Flavored Markdown?

Reply:
The current behavior *should* be consistent with GFM - fenced code blocks with a language specifier (without the {}'s) are interpreted correctly:

````
```python
def my_pythonic_fenced_code_block():
    pass
```
````
The `{code-block}` directive is useful if you need additional functionality, such as captions/labels.

Reply:
Ah ok, good, that makes a lot of sense. =)